views/repo_list.pug

```diff
@@ -1,9 +1,25 @@
 h1
   Github Repositories
 
-.git-repo-areas
+.git-repo-areas(ng-controller="ListItemCtrl")
```

Comment:
actually calling a function once on initialization for many controllers are preferable compared to calling `$scope.updateOn` and `$scope.languageColors` functions multiple time in its life time.